src/types.py

```diff
@@ -309,7 +320,7 @@ def get_parameter_size_align_bytes(self) -> Tuple[int, int]:
         data = self.data()
         if self.is_struct():
             assert data.struct is not None
-            return data.struct.size, data.struct.align
+            return data.struct.min_size(), data.struct.align
```

Comment:
This is rather sketchy. Should probably throw a DecompFailure(f"Struct parameter of incomplete type {data.struct}") instead, not that it's likely to ever come up in practice